### ISSUE TITLE
when is the UDF invoked?

### ISSUE THREAD

**Issue by huanggl99** [xlwings/xlwings#2317]
#### Describe your issue (incl. Traceback!)
when is the UDF called?
i did a simple UDF like =test(A1)
supposely the UDF is called and recalculated once cell A1's value has changed. but i test it is not the case. actually, once every cell value other than A1 in the sheet change, the UDF will be called, that will be an issue because those calculations are not neccessary as A1 doens't change

```python
# Your traceback here

```

#### Include a minimal code sample to reproduce the issue (and attach a sample workbook if required!)

```python
# Your code here

```

**Comment by fzumstein** [comment]
No, this is only the case if you mark the UDF as volatile: https://docs.xlwings.org/en/latest/api/udf_decorators.html#xlwings.func
